week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,12 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+var total = numberOfStudents + numberOfMentors;
+var studentPerc = numberOfStudents * 100 / total;
+var mentorsPerc = numberOfMentors *100 / total;
+var roundStudentPerc = Math.round(studentPerc);
+var roundMentorsPerc = Math.round(mentorsPerc);
```

Comment:
Can you think of a way to use a function to make this more DRY (don't repeat yourself)?